ietf-document/draft-quic-4-sat/draft-kuhn-quic-4-sat-04.txt

```diff
@@ -0,0 +1,728 @@
+
+
+   o  Asymmetry : when the links are asymmetric, for various reasons,
+      the sizing of the return link may induce modifications of the
+      transport level acknowledgement traffic.
+
+3.  TCP Split Solution
+
+   High BDP networks commonly break the TCP end-to-end paradigm to adapt
+   the transport protocol.  Splitting TCP allows adaptations to this
+   specific use-case and assessing the issues discussed in section
+   Section 2.  Satellite communications commonly deploy Performance
+   Enhancement Proxy (PEP) for compression, caching and TCP acceleration
+   services [RFC3135].  Their deployment can result in 50% page load
+   time reduction in a SATCOM use-case [ICCRG100].
+
+   [NCT13] and [RFC3135] describe the main functions of SATCOM TCP split
+   solutions.  Shortly, for traffic originated at a gateway to an
+   endpoint connected via a satellite terminal, the TCP split intercepts
+   TCP SYN packets to act on behalf of the endpoint and adapt the data
+   rate transmission to the SATCOM scenario.  The split solution
+   specifically tunes the TCP parameters to the context (latency,
+   available capacity) of each link.  When a PEP is used on each side of
+   the satellite link, a protocol other than TCP, optimized for the
+   satellite link, may be used.  The tuning can be achieved using a
+   priori information about the satellite system and/or by measuring the
+   properties of the network segment that includes the satellite system.
+   Split connections also allow for recovery from packet losses that is
+   local to the part of the connection on which the packet losses
+   occurred.  This eliminates the need for end-to-end recovery of lost
+   packets.
+
+   One important advantage of a TCP split solution is that it does not
+   require any end-to-end modifications and is independent for both
+   client and server sides.  That being said, this comes with a
+   drawback: TCP splitters often are unable to track the most recent
+   end-to-end improvements in protocol mechanisms (e.g., RACK, ECN, TCP
+   Fast Open support) contributing to ossification of the transport
+   system.  The methods configured in the split proxy usually continue
+   to be used until a split solution is finally updated.  This can
+   delay/negate the benefit of any end-to-end improvements.
+
+4.  Regression tests
+
+   This section proposes regression tests for QUIC.  We define by:
+
+   o  Download path: from Internet to host
+
+   o  Upload path: from the host to Internet
+
+
+
+
+Kuhn, et al.             Expires August 31, 2020                [Page 5]
+
+Internet-Draft                 QUIC 4 SAT                  February 2020
+
+
+4.1.  Small public satellite broadband access
+
+   The characteristics of the architecture under test are the following:
+
+   o  Satellite downlink path: 10 Mbps
+
+   o  Satellite uplink path: 2 Mbps
+
+   o  No emulated packet loss
+
+   o  RTT: 650 ms
+
+   o  Buffer size : BDP
+
```

Comment:
Buffer = BDP is perhaps OK. On the other hand I think the variable return path often demands a higher BDP ... however I would be really content with a reference scenario that says 1 BDP, because it is simple and clear.